go.mod

```diff
@@ -124,3 +124,5 @@ require (
 	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
 	sigs.k8s.io/yaml v1.1.0
 )
+
+go 1.13
```

Comment:
Your machine added this line behind the scenes, because you have go 1.13 installed. However if we merge it, we hav a `go.mod` with go 1.13, a `.circleci/config.yml` with go 1.12 and a `Dockerfile` with go 1.11.
I recommend to remove this line from your PR to not conflict with #26, and then after #26 is merged, everything is up to date on the same version 😊